src/modules/Layout/Header/Header.scss

```diff
@@ -10,7 +10,7 @@
   &__col {
     align-self: center;
   }
-
+  
```

Comment:
looks like trailing spaces. @ivanzusko can you please set up your editor to remove them automatically